src/App.tsx

```diff
@@ -19,55 +50,55 @@
               placeholder="Enter a part of the name"
               className="input"
               data-cy="search-input"
+              onChange={handleQuery}
```

Comment:
Your event handler function `handleQuery` doesn't follow the naming conventions for event handlers in React. Consider renaming it to `onQueryChange`.